tests/kafkatest/services/connect.py

```diff
@@ -52,7 +52,7 @@ class ConnectServiceBase(KafkaPathResolverMixin, Service):
             "collect_default": True},
     }
 
-    def __init__(self, context, num_nodes, kafka, files):
+    def __init__(self, context, num_nodes, kafka, files=[]):
```

Comment:
Are we adding these defaults for a reason? I don't see how these are being taken advantage of and in general, object defaults are dangerous since a single instance is shared across all calls (so, e.g., if a list is mutated, all subsequent calls will see the modified value).